.github/workflows/dependencies.yml

```
@@ -87,6 +87,22 @@ jobs:
           path: |
             cargo-deny.log
             
+  ban-legacy-agent-path:
+    name: Ban legacy .agent path
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)